seatunnel-e2e/seatunnel-connector-v2-e2e/connector-iceberg-hadoop3-e2e/src/test/java/org/apache/seatunnel/e2e/connector/iceberg/hadoop3/IcebergSourceIT.java

```diff
@@ -124,6 +126,7 @@ public void testIcebergSource(TestContainer container) throws IOException, Inter
     }
 
     private void initializeIcebergTable() {
+        FileUtil.fullyDelete(new File(CATALOG_DIR));
```

Comment:
Would this be a problem if it wasn't completely deleted?

Reply:
Test cases can interfere with each other